cpp/src/copying/contiguous_split.cu

```diff
@@ -94,7 +94,7 @@ struct dst_buf_info {
   int num_rows;  // # of rows (which may be different from num_elements in the case of validity or
                  // offset buffers)
   int src_row_index;  // row index to start reading from from my associated source buffer
-  int dst_offset;     // my offset into the per-partition allocation
+  size_t dst_offset;  // my offset into the per-partition allocation
```

Comment:
```suggestion
  std::size_t dst_offset;  // my offset into the per-partition allocation
```

Reply:
Make this change throughout.

Reply:
There's a lot of other size_t usage in this file.  Should I change those too?

Reply:
Up to you. I'd think that'd be easiest since you can just do find/replace all.